packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -79,6 +79,20 @@ export abstract class TableColumn<
     @observable
     public internalMinPixelWidth = defaultMinPixelWidth;
 
+    /**
+     * @internal
+     * Whether or not a column can be used to group rows by
+     */
+    @observable
+    public internalGroupingDisabled = false;
+
+    /**
+     * @internal
+     * The order in which to group column values by
```

Comment:
This _is_ an internal API, so I'm not sure how much we gain with a verbose description. That said, I'm fine with going with the suggestion unless someone feels otherwise.